### PR TITLE
Added documentation for the rpi_pfio platform.

### PR DESCRIPTION
**Pull request by basschipper**
**Description:**
Support for the PiFace Digital I/O module (Raspberry Pi add on board)
This is basically a port of the rpi_gpio component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7494

### CODE REVIEW

File: source/_components/binary_sensor.rpi_pfio.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: "PiFace Digital I/O Binary Sensor"
+description: "Instructions how to integrate the PiFace Digital I/O module into Home Assistant as a binary sensor."
+date: 2016-05-08 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: raspberry-pi.png
+ha_category: Binary Sensor
+ha_release: 0.44.3
```
Comment (Landrash) [inline source/_components/binary_sensor.rpi_pfio.markdown]: .45 will be the next release.

File: source/_components/binary_sensor.rpi_pfio.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+The `rpi_pfio` binary sensor platform allows you to read sensor values of the [PiFace Digital I/O](https://www.element14.com/community/docs/DOC-69001) .
```
Comment (Landrash) [inline source/_components/binary_sensor.rpi_pfio.markdown]: Could the link be pointed to the manufacture instead of Element 14?

**Review by Landrash — CHANGES_REQUESTED** [review]
Looks really good and only needed change is to update the release number before release.

If a better link is available I would recommend linking to the maker of the product instead of Element 14.

File: source/_components/binary_sensor.rpi_pfio.markdown (hunk 2 above)
Comment (basschipper) [inline source/_components/binary_sensor.rpi_pfio.markdown]: Element14 seems to be the manufacturer of the PiFace.

**Review by Landrash — APPROVED** [review]
Good to go and can be merged when parent PR is merged.

File: source/_components/binary_sensor.rpi_pfio.markdown (hunk 2 above)
Comment (Landrash) [inline source/_components/binary_sensor.rpi_pfio.markdown]: Oh. Well then my bad :)

**Comment by roland1970** [comment]
Hello Bas ,Is it possible to add more then 1 piface?
I mean to add the address-rule so that we can use more pifaces.
Or can you tell me what I must change make that possible .

**Comment by klaasnicolaas** [comment]
Please don't comment on old pull requests. You can create an issue here instead: https://github.com/home-assistant/core/issues/new/choose

**Pull request merged**